docs/ibmcloud_help.md

```diff
@@ -3,12 +3,101 @@
 **Table of Contents**  *generated with [DocToc](https://github.com/thlorenz/doctoc)*
 
 - [IBM Cloud](#ibm-cloud)
+  - [Inspect files `cluster.yaml` and `machines.yaml`.](#inspect-files-clusteryaml-and-machinesyaml)
   - [How to set correct OS reference code in `machines.yaml`?](#how-to-set-correct-os-reference-code-in-machinesyaml)
-  - [how to set `dataCenter` value in `machines.yaml`?](#how-to-set-datacenter-value-in-machinesyaml)
+  - [How to set `dataCenter` value in `machines.yaml`?](#how-to-set-datacenter-value-in-machinesyaml)
 
 <!-- END doctoc generated TOC please keep comment here to allow auto update -->
 
 # IBM Cloud
+## Inspect files `cluster.yaml` and `machines.yaml`.
+
+cluster.yaml
```

Comment:
this is `ibmcloud_help.md` which is targeted to IBM Cloud related knowledge.  `cluster.yaml` has no relationship with IBM cloud but for cluster API.

Reply:
OK, Is the trouble-shooting.md good to tell user about the inconsistent cluster could cause failure?
Seems that new user may hit this failure. No error message about it.